### PR TITLE
NMS-12985: Fixed another typo in VmwareAssetsMigratorOffline

### PR DESCRIPTION
**Pull request by christianpape**
* JIRA: https://issues.opennms.org/browse/NMS-12985

No test included yet. At the moment I have no idea how a test could look like, because the Migrator needs a certain database structure.

### CODE REVIEW

**Review by RangerRick — APPROVED** [review]
guess I missed one  :D

File: core/upgrade/src/test/java/org/opennms/upgrade/implementations/VmwareAssetsMigratorOfflineIT.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        this.databasePopulator.populateDatabase();
+    }
+
+    @Override
+    public void setTemporaryDatabase(TemporaryDatabase database) {
+        this.temporaryDatabase = database;
+    }
+
+    @Test
+    public void foo() throws Exception {
```
Comment (fooker) [inline core/upgrade/src/test/java/org/opennms/upgrade/implementations/VmwareAssetsMigratorOfflineIT.java]:
```suggestion
    public void bar() throws Exception {
```

File: core/upgrade/src/test/java/org/opennms/upgrade/implementations/VmwareAssetsMigratorOfflineIT.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+})
+@JUnitConfigurationEnvironment
+@JUnitTemporaryDatabase(reuseDatabase = false)
+public class VmwareAssetsMigratorOfflineIT implements TemporaryDatabaseAware<TemporaryDatabase> {
+
+    @Autowired
+    DatabasePopulator databasePopulator;
+
+    @Autowired
+    TransactionOperations transactionOperations;
```
Comment (fooker) [inline core/upgrade/src/test/java/org/opennms/upgrade/implementations/VmwareAssetsMigratorOfflineIT.java:71]: Use `SessionUtils` instead.

**Review by fooker — CHANGES_REQUESTED** [review]

**Review by fooker — APPROVED** [review]

**Comment by fooker** [comment]
@RangerRick is this the right target branch or should this go into `release-27.x` instead?

**Comment by RangerRick** [comment]
Oh yeah definitely 27

**Comment by opennms-bamboo** [comment]
Welcome CI/CD System (cicd-system@opennms.com).

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username CI/CD System (cicd-system@opennms.com).
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Comment by fooker** [comment]
@christianpape I've updated the target branch, but you have to rebase.

**Review by RangerRick — APPROVED** [review]
yay, tests

**Pull request merged**